python/neuroglancer/ingest/ingest.py

```diff
@@ -57,7 +60,7 @@ def upload_info(self):
             size=self._volume.shape,
             voxel_size=self._volume.resolution,
             dimensions_to_downsample=[0, 1, 2],
-            max_downsampled_size=np.max(self._volume.shape / self._chunk_size * self._neuroglancer_chunk_size)
+            max_downsampling=int(reduce(operator.mul, self._chunk_size / self._neuroglancer_chunk_size )),
```

Comment:
Max downsampling is always a positive number so floor and int are equivalent. A comment is  a good idea.